deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -33,7 +33,7 @@ spec:
 
     ### Pre-requisites
     * A Red Hat OpenShift subscription
-    * OCP 4.6.8+ cluster running on Azure, AWS or vSphere configured with hybrid OVN Kubernetes networking
+    * OKD/OCP 4.7 cluster running on Azure, AWS or vSphere configured with hybrid OVN Kubernetes networking
```

Comment:
Product WMCO is only for OCP. So `OCP 4.7` is sufficient.

Reply:
fixed